### PR TITLE
[WIP] Add Info

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/user-api/pull/577

### CODE REVIEW

File: thoth/management_api/api_v1.py [hunk 1]
```diff
@@ -39,6 +39,21 @@
 _OPENSHIFT = OpenShift()
 
 
+def get_info():
+    """Get information about Thoth deployment."""
+    return {
+        "deployment_name": os.getenv("THOTH_DEPLOYMENT_NAME"),
+        "version:": os.getenv("OPENSHIFT_BUILD_REFERENCE", "@dev"),
```
Comment (goern) [inline thoth/management_api/api_v1.py]: @fridex where do we define "the thoth version"? @harshad16

File: thoth/management_api/api_v1.py [hunk 2]
```diff
@@ -39,6 +39,21 @@
 _OPENSHIFT = OpenShift()
 
 
+def get_info():
+    """Get information about Thoth deployment."""
+    return {
```
Comment (goern) [inline thoth/management_api/api_v1.py]: are all these ENV VAR define for a deployment of management-api?

File: thoth/management_api/api_v1.py (hunk 1 above)
Comment (fridex) [reply thoth/management_api/api_v1.py]: User API sends it in headers. IMHO, this `OPENSHIFT_BUILD_REFERENCE` can be removed.

https://github.com/thoth-station/user-api/blob/6ce5680a5b2f07391e19e31e3dacf9a8179e4773/thoth/user_api/openapi_server.py#L141-L145

Comment (bissenbay) [reply thoth/management_api/api_v1.py]: Removed!

File: thoth/management_api/api_v1.py [hunk 3]
```diff
@@ -39,6 +39,20 @@
...
+        "deployment_name": os.getenv("THOTH_DEPLOYMENT_NAME"),
+        "s3_endpoint_url": os.getenv("THOTH_S3_ENDPOINT_URL"),
+        "dgraph_host": os.getenv("GRAPH_SERVICE_HOST"),
```
Comment (fridex) [inline thoth/management_api/api_v1.py]:
```patch
-        "dgraph_host": os.getenv("GRAPH_SERVICE_HOST"),
+        "knowledge_graph_host": os.getenv("KNOWLEDGE_GRAPH_HOST"),
```

It has changed recently:

https://github.com/thoth-station/management-api/blob/a52634dc020491ab942c632c3e50d8799d3e87b8/openshift/deployment-template.yaml#L201

Comment (bissenbay) [reply thoth/management_api/api_v1.py]: Good catch, thanks @fridex !

**Pull request merged**